content/docs/atom-guide/look-dev/six-point-lighting-tutorial.md

```diff
@@ -0,0 +1,439 @@
+
+1. Open `SixPointLighting_TexturePackEnum.lua`. Notice the two functions `GetMaterialPropertyDependencies()` and `ProcessEditor()`. `GetMaterialPropertyDependencies()` gets the value of a property of a material. `ProcessEditor()` can then use the property values to enable and disable visibility of the properties in the Material Editor.
+2. Following how `sixPointLighting.TLRB`'s visibility is enabled and disabled, enable and disable the other texture options as appropriate:
+   ```lua
+   if(texturePackMode == TexturePackMode_TpLftRtBt_FrBck) then
+      -- TopLeftRightBack is the first texture, FrontBack is the second. Disable RightLeftTop and BottomBackFront.
+      context:SetMaterialPropertyVisibility("sixPointLighting.TLRB", MaterialPropertyVisibility_Enabled)
+      context:SetMaterialPropertyVisibility("sixPointLighting.FB", MaterialPropertyVisibility_Enabled)
+      context:SetMaterialPropertyVisibility("sixPointLighting.RLT", MaterialPropertyVisibility_Hidden)
+      context:SetMaterialPropertyVisibility("sixPointLighting.BBF", MaterialPropertyVisibility_Hidden)
+   elseif(texturePackMode == TexturePackMode_RtLftTp_BtBckFr) then
+      -- RightLeftTop is the first texture, BottomBackFront is the second. Disable TopLeftRightBack and FrontBack.
+      context:SetMaterialPropertyVisibility("sixPointLighting.TLRB", MaterialPropertyVisibility_Hidden)
+      context:SetMaterialPropertyVisibility("sixPointLighting.FB", MaterialPropertyVisibility_Hidden)
+      context:SetMaterialPropertyVisibility("sixPointLighting.RLT", MaterialPropertyVisibility_Enabled)
+      context:SetMaterialPropertyVisibility("sixPointLighting.BBF", MaterialPropertyVisibility_Enabled)
+   end
+   ```
+
+## Make a six-point lighting material
+In order to see the Material Editor changes take place, make a six-point lighting material.
+
+1. Open the **Material Editor**, and make a new material with the six-point lighting material type.
+   
+2. Find the **Six Point Lighting** properties in the **Inspector**.
+   
+3. Notice how the default **Texture Pack Mode** is `TpLftRtBt_FrBck`. The two properties below that correspond to this texture pack mode, and the properties for the other texture pack mode are hidden. 
+
+4. Select `RtLftTp_BtBckFr` for the **Texture Pack Mode** and observe how the properties change.
+
+5. Set the following properties accordingly:
+    * **Six Point Lighting**
+      * **Texture Pack Mode**: `RtLftTp_Bt_BckFr`
+      * **Right Left Top**: `SmokeBall01_6Way_RLT_8x8.png`
+      * **Bottom Back Front**: `SmokeBall01_6Way_BBF_8x8.png`
+      * **Rows in Flipbook**: `8.0`
+      * **Columns in Flipbook**: `8.0`
+    * **Base Color**
+      * **Texture**: `SmokeBall01_ColorCC_8x8.png`
+      * **Use Texture**: Disabled
+         {{< note >}}
+         Using the texture as the base color will discolor how the material looks, but the texture's alpha channel will be used for the opacity.
```

Comment:
I see, the two phrases I wrote don't relate at all. I replaced the sentence with this:

>  {{< note >}}
>  You don't want to use the texture as the base color because it will discolor the material. However, setting the base color texture property is necessary to use the texture's alpha channel for the opacity.
>  {{< /note >}}